sdk/data_tables/src/lib.rs

```diff
@@ -63,8 +63,6 @@ async fn main() -> azure_core::Result<()> {
 
 */
 
-#[macro_use]
-extern crate log;
```

Comment:
@heaths, this is what required the inclusion of `tracing::trace` and the like in a handful of places.

Rather than import _all_ of the macros from the tracing crate implicitly, this PR move to explicitly using the intended macros in the intended locations.